src/gui/systray.cpp

```diff
@@ -79,6 +80,14 @@ Systray::Systray()
         }
     );
 
+#ifndef Q_OS_MAC
+      auto contextMenu = new QMenu();
```

Comment:
Indentation seems busted in that block.

Also the request from @jancborchardt was to have: "Open Main Dialog", "Settings" and "Quit" in that menu.

Reply:
I am not sure if I did this write because now I noticed a function in owncloudgui that is called slotOpenMainDialog that actually just shows the systray menu.

Reply:
Now open main dialog open the main window and settings actually shows the settings panel.

Reply:
What is the main dialog? Is it the new systray menu or the mains window with the settings and network?
https://github.com/nextcloud/desktop/blob/be4ba031ec0c7265bbc53938969b2b47d551caeb/src/gui/application.cpp#L665

I would say this call after the user created his account needs to open the window with the settings not the new systray menu.

Reply:
> What is the main dialog? Is it the new systray menu or the mains window with the settings and network?

It is the new window you get when clicking on the systray. We call that the "main dialog" now, the other one being the "settings dialog".

> https://github.com/nextcloud/desktop/blob/be4ba031ec0c7265bbc53938969b2b47d551caeb/src/gui/application.cpp#L667
> 
> I would say this call after the user created his account needs to open the window with the settings not the new systray menu.

Dunno... I don't think I mind either way. :-)

Reply:
Yup. Undid it.

Reply:
> Also the request from @jancborchardt was to have: "Open Main Dialog", "Settings" and "Quit" in that menu

Sorry, I just wrote that in the chat based on what the things should do, not the exact wording. It should of course reflect the wording we use in the main dialog:
- Open main dialog (only first word capitalized
- Settings
- Quit Nextcloud

Since we had the point brought up in the chat as well, we could take the opportunity to change it to "Exit Nextcloud" instead, here as well as in the main dialog.

Reply:
Done.